openfe/protocols/openmm_afe/equil_solvation_afe_method.py

```diff
@@ -521,7 +521,7 @@ def _create(
         self,
         stateA: ChemicalSystem,
         stateB: ChemicalSystem,
-        mapping: Optional[Dict[str, gufe.ComponentMapping]] = None,
+        mapping: list[gufe.ComponentMapping],
```

Comment:
As discussed on the gufe PR, I would very much like this not to be how we do things going forward - it's very unintuitive from a Pythonic sense.